modules/vpc/user_management.yml.tmpl

```diff
@@ -0,0 +1,49 @@
+---
+Consul:
+  Server: 127.0.0.1:8500
+  Namespace: "environments/${environment}/global/nubis-users"
+
+AWS:
+  Region: "${region}"
+  SMTPFromAddress: "${smtp_from_address}"
+  SMTPUsername: "${smtp_username}"
+  SMTPPassword: "${smtp_password}"
+  SMTPHostname: "${smtp_host}"
+  SMTPPort:     "${smtp_port}"
+  AWSIgnoreUserList:
+    - nubis-bootstrap
+    - datadog
+    - gozer
+    - limed
+  AWSIgnorePathList:
+    - /
```

Comment:
Curious as to how the AWSIgnorePathList mechanism functions, the way I read this, wouldn't '/' ignore pretty much everything ? Or does that just mean to ignore users without a specific path set? i.e. just username, path=/ ?